docs/developers.md

```diff
@@ -0,0 +1,93 @@
+  [go-ethereum/rpc](https://pkg.go.dev/github.com/ethereum/go-ethereum/rpc)
+  client to access Starknet.
+
+If you wonder which one to use, the short answer is it depends. However, Caigo
+provide an access to both and tries to share a common interface between the 2.
+Be careful that when you are doing something on the project, you should pay
+attention to the 2 interfaces, even if the implementations are specific.
+
+## Where do I find resources about the protocol specifications?
+
+There are several efforts to document how the API are working. You should
+know them and you can contribute to the associated projects:
+
+- [starknet-edu postman reference](https://www.postman.com/starknet-edu/workspace/starknet-edu/collection/20082312-a5291c43-a4e5-4a6d-9c51-125c6acd3b41?ctx=documentation)
+  is a project that lists all the existing API and provides some examples
+  of how to use them.
+- There is no true specification of the API we are aware of, however the
+  implementationis opensource and can be read from the
+  starkware-libs/cairo-lang
+  [api/gateway directory](https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware/starknet/services/api/gateway)
+- Starknet `openrpc` specification is available in the
+  `starkware-libs/starknet-specs` project; check the content of
+  [this directory](https://github.com/starkware-libs/starknet-specs/blob/master/api)
+
+If you want to go deeper into the protocol, there are other interesting
+resources available, check:
+
+- [a set of examples](https://github.com/eqlabs/pathfinder/blob/main/crates/pathfinder/rpc_examples.sh)
+  from the eqlabs/pathfinder project that provides RPC tests with the project.
+  Note that the [go-ethereum] implementation of openrpc only support positional
+  arguments and, some of these examples must be changed into positional calls.
+- `#🦦| caigo` channel in the discord from Starknet.
+
+## How to interact with accounts?
+
+An important part of interacting with Starknet consists in interacting
+accounts. The reason is that you need go through an account to run transactions,
+including to create a contract from a class. That is why you should understand
+what is an account is and how to interact with it.
+
+The devil living in the details, there are specificities associated with
+account implementations. For instance, you might find that one account does not
+support the same signature as an other. To start with account, read:
+
+- [Starknet Account Abstraction Model - Part 1](https://community.starknet.io/t/starknet-account-abstraction-model-part-1/781)
+- [Starknet Account Abstraction Model - Part 1](https://community.starknet.io/t/starknet-account-abstraction-model-part-2/839)
+- Openzeppelin Cairo contracts that includes an account
+  [implementation](https://github.com/OpenZeppelin/cairo-contracts/tree/main/src/openzeppelin/account)
+- The argent-x contract
+  [implementation](https://github.com/argentlabs/argent-contracts-starknet)
+
```

Comment:
Do we want a link to the account abstraction tutorial ? https://github.com/starknet-edu/starknet-accounts

Reply:
Added! I did not know that resource 💯